src/containers/Tables/BidTable/BidTable.utils.ts

```diff
@@ -0,0 +1,15 @@
+export const handleDomainNameWidth = (dimensions: number) => {
+	const isMobileM = dimensions <= 400;
+	const isMobileL = dimensions <= 481;
```

Comment:
Is this specific for BidTable or can we make it generic for usage across app.

Reply:
We could make it generic but this was more tailored for BidTable and the layout of its elements. This was more of a quick fix with domain name truncation, I believe the handling of domain names is in the Todo backlog and will be a task for the near future.

Reply:
sounds good. Approving it.